packages/roosterjs-editor-plugins/lib/ContentEdit/features/tableFeatures.ts

```diff
@@ -98,7 +92,9 @@ export const UpDownInTable: ContentEditFeature = {
 function cacheGetTableCell(event: PluginEvent, editor: Editor): HTMLTableCellElement {
     return cacheGetEventData(event, 'TABLECELL_FOR_TABLE_FEATURES', () => {
         let pos = editor.getFocusedPosition();
-        let firstTd = editor.getElementAtCursor('TD,TH,LI', pos.node);
-        return getTagOfNode(firstTd) == 'LI' ? null : (firstTd as HTMLTableCellElement);
+        let firstTd = pos && editor.getElementAtCursor('TD,TH,LI', pos.node);
+        return (
+            firstTd && (getTagOfNode(firstTd) == 'LI' ? null : (firstTd as HTMLTableCellElement))

```

Comment:
When firstTd is null, getTagOfNode(firstTd) will return null. So the function will return null correctly.

Reply:
You're right, but I was thinking about strict mode, and the current signature of `getTagOfNode` takes in a non null value. Granted, could expand that check to also take in a null value, or remove this check entirely. Up to you.

Reply:
Ok. I can approve the change since it can make the code more clear.